src/base-element.js

```diff
@@ -172,6 +172,15 @@ export class BaseElement {
     return this.win;
   }
 
+  /**
+   * Returns the associated ampdoc. Only available when `buildCallback` and
```

Comment:
nit: mention it throws if accessed before buildCallback (I had to look at implementation to see if I get null or exception, so worth documenting)